### ISSUE TITLE
Error 403 upon login attempt

### ISSUE THREAD

**Issue by saisenberg** [matthew1232/stockx-api#32]
Hey Matthew, you may already be aware of this, but it looks like StockX has updated their login process within the last couple weeks or so. I've been getting a 403 error upon trying to log in with the following error message. Looks like there's some additional captcha being triggered. 

`{"appId":"PX16uD0kOF","jsClientSrc":"/16uD0kOF/init.js","firstPartyEnabled":true,"vid":"","uuid":"c6225e53-1c92-11ec-ace3-77726755437a","hostUrl":"/16uD0kOF/xhr","blockScript":"/16uD0kOF/captcha/captcha.js?a=c&u=c6225e53-1c92-11ec-ace3-77726755437a&v=&m=0"}`

A HUGE thank you for working on this repo and keeping it updated!!

**Comment by AceHack** [comment]
I'm getting the exact same error, any luck?

Error: Status code error 403 - Response: {"appId":"PX16uD0kOF","jsClientSrc":"/16uD0kOF/init.js","firstPartyEnabled":true,"vid":"","uuid":"712876af-3b52-11ec-8fe5-677768516a53","hostUrl":"/16uD0kOF/xhr","blockScript":"/16uD0kOF/captcha/captcha.js?a=c&u=712876af-3b52-11ec-8fe5-677768516a53&v=&m=0"}

**Comment by alexchiriac** [comment]
I have the same issue. Has anyone found a fix? @matthew1232 since you worked on the original version, any idea how we might be able to fix this ourselves in case you don't have time to update the project?

**Comment by LyesEpitech** [comment]
Same issue here :(

**Comment by abhishek7d** [comment]
same issue 403 when i am trying to login

**Comment by sssst9s** [comment]
Its probably due to the captcha system wich not anymore allow you to get the login details, there is way to fix it but idk with his package

**Comment by J0ker98** [comment]
I made a fork of this repository with the correct adjustments for a working login. Feel free to contribute here: https://github.com/J0ker98/stockx-api

**Comment by sssst9s** [comment]
@J0ker98 is it working the same as matthew's code ?

**Comment by J0ker98** [comment]
> @J0ker98 is it working the same as matthew's code ?

Yes, it works the same way but contains fixes and additional features

**Comment by dennisblack777** [comment]
> > @J0ker98 is it working the same as matthew's code ?
> 
> Yes, it works the same way but contains fixes and additional features

Also with your fork comes a 403 error to confirm that you are not a bot. :(

**Comment by J0ker98** [comment]
> > > @J0ker98 is it working the same as matthew's code ?
> > 
> > 
> > Yes, it works the same way but contains fixes and additional features
> 
> Also with your fork comes a 403 error to confirm that you are not a bot. :(

It should be working as soon as you are not making an eccessive amount of requests

**Comment by kewuamigo** [comment]
Also get 403 with your fork @J0ker98 I try with different residential IP and I get 403 with first req
